atacworks/dl4atac/utils.py

```diff
@@ -236,7 +236,7 @@ def gather_files_from_cmdline(input, extension):
             res = [path]
         elif os.path.isdir(path):
             paths = [os.path.join(path, f)
-                     for f in os.listdir(path)]
+                     for f in sorted(os.listdir(path))]
```

Comment:
Can we add to the help messages in `scripts/cmd_args.py` to say that the files will be read in sorted order?

Reply:
done